src/pages/library/datasetBuilder/DatasetBuilder.test.ts

```diff
@@ -246,11 +256,86 @@ describe('DatasetBuilder', () => {
     const user = userEvent.setup();
     const onStateChange = jest.fn();
     render(
-      h(DatasetBuilderContents, { cohorts: [], conceptSets: [], onStateChange, dataset: dummyDatasetDetails('id') })
+      h(DatasetBuilderContents, {
+        updateCohorts: jest.fn(),
+        updateConceptSets: jest.fn(),
+        cohorts: [],
+        conceptSets: [],
+        onStateChange,
+        dataset: dummyDatasetDetails('id'),
+      })
```

Comment:
You could use a helper function here to avoid code duplication. For example, see `showCohortEditor()` in `CohortEditor.test.ts`.